src/AlgebraicGeometry/Miscellaneous/basics.jl

```diff
@@ -45,7 +45,7 @@ function weights(P::ProjSpc)
   return Int[x[1] for x = P.Rx.d]
 end
 
-function isweighted(P::ProjSpc)
+function is_weighted(P::ProjSpc)
   return !all(x->isone(x[1]), P.Rx.d)
```

Comment:
I find this method problematic -- why is a projective space in which all weights are 1 "not weighted" ?